packages/xo-web/src/xo-app/backup-ng/restore/index.js

```diff
@@ -131,7 +131,7 @@ export default class Restore extends Component {
     jobs = this.props.jobs
   ) => {
     const remotes = keyBy(filter(_remotes, { enabled: true }), 'id')
-    const backupsByRemote = await listVmBackups(toArray(remotes))
+    const backupsByRemote = await listVmBackups(toArray(remotes), true)
```

Comment:
Nope, only when the user click on the button, otherwise we completely loose the benefits of the cache.